reference/5.1/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -0,0 +1,190 @@
+
+Integer literals can be written in decimal or hexadecimal notation. Hexadecimal
+literals are prefixed with "0x" to distinguish them from decimal numbers.
+
+Integer literals can have a type suffix and a multiplier suffix.
+
+| Suffix |       Meaning       |
+| ------ | ------------------- |
+| l      | long data type      |
+| kb     | kilobyte multiplier |
+| mb     | megabyte multiplier |
+| gb     | gigabyte multiplier |
+| tb     | terabyte multiplier |
+| pb     | petabyte multiplier |
+
+The type of an integer literal is determined by its value, the type suffix, and
+the numeric multiplier suffix.
+
+For an integer literal with no type suffix:
+
+- If the value can be represented by type `[int]`, that is its type.
+- Otherwise, if the value can be represented by type `[long]`, that is its
+  type.
+- Otherwise, if the value can be represented by type `[decimal]`, that is its
+  type.
+- Otherwise, it is represented by type `[double]`.
+
+For an integer literal with a type suffix:
+
+- If the type suffix is `u` and the value can be represented by type `[int]`
+  then its type is `[int]`.
+- If the type suffix is `u` and the value can be represented by type `[long]`
+  then its type is `[long]`.
+- If its value can be represented by type specified then that is its type.
+- Otherwise, that literal is ill formed.
+
+## Real literals
+
+Real literals can only be written in decimal notation. This notation can
+include fractional values following a decimal point and scientific notation
+using an exponential part.
+
+The exponential part includes an 'e' followed by an optional sign (+/-) and a
+number representing the exponent. For example, then literal value `1e2` equals
+the numeric value 100.
+
+Real literals can have a type suffix and a multiplier suffix.
+
+| Suffix |       Meaning       |
+| ------ | ------------------- |
+| d      | decimal data type   |
+| kb     | kilobyte multiplier |
+| mb     | megabyte multiplier |
+| gb     | gigabyte multiplier |
+| tb     | terabyte multiplier |
+| pb     | petabyte multiplier |
+
+There are two kinds of real literal: double and decimal. These are indicated by
+the absence or presence, respectively, of decimal-type suffix. PowerShell does
+not support a literal representation of a `[float]` value. A double real
+literal has type `[double]`. A decimal real literal has type `[decimal]`.
+Trailing zeros in the fraction part of a decimal real literal are significant.
+
+If the value of exponent-part's digits in a `[double]` real literal is less
+than the minimum supported, the value of that `[double]` real literal is 0. If
+the value of exponent-part's digits in a `[decimal]` real literal is less than
+the minimum supported, that literal is ill formed. If the value of
+exponent-part's digits in a `[double]` or `[decimal]` real literal is greater
+than the maximum supported, that literal is ill formed.
+
+> [!NOTE]
+> The syntax permits what a double real literal to have a long-type suffix.
```

Comment:
`what` feels out of place in this sentence... can you review it is as intended?

Reply:
Fixed